style.css

```diff
@@ -0,0 +1,168 @@
+body {
+  margin: 0;
+  padding: 0;
+  background-image: url("images/apple-back.jpg");
+  background-repeat: no-repeat;
+  background-size: 1400px 1166px;
+  background-position: 0 -700px;
+  overflow-x: hidden;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
```

Comment:
**Hint: I notice this is where all the changes happen. When I defined a body max-width and also changed the background-size width, it fixed the background cover problem. Probably you can play around with that. Here is the code I added/ changed.**
```
body {
 max-width: 100%;
background-size: 100% 1166px;
}
```